Add minutes for TSC April 2024 call (#87)
servo/project: governance/tsc/tsc-2024-04-29.md

martin: there's a couple of interesting edge cases, rust-harfbuzz is one of them

martin: it's a low level API, which servo uses; and a high level one, which servo doesn't use

martin: and there's an official harfbuzz that provides the high level API

martin: Behdad has say they'll be ok adopting things we cannot maintain anymore, would it make sense to give up ownership of these crates

martin: one option is give up on the high level API we're not using, or give up in both of those and then the harfbuzz project would be responsibly also for maintaining it

martin: another option could be transfer both, the issue might be if Servo needs a new version at some point

manish: I'm happy to at least maintenance mode maintain the crate. I think that could work.

manish: If we want harfbuzz-sys to be in Servo and to have one of them maintained by a broader team.

martin: that raises another question, harfbuzz has its own high level bindings, not sure if they have a different philosophy

nico: the harfbuzz one hasn't been updated for 9 months

> martin: our looks better now but there was a big gap anyway

Reviewer: ```suggestion
martin: Ours look better now, but that was only after some recent work making changes. Before that there was a gap of a few years.
```

> martin: ???

Author: @mrobinson dunno if you can guess what was the question/comment here.

Author: Ok, no worries.

manish: I like harfbuzz-rs, the crate we have, it's useful; if there's not a lot of effort to maintain it; there's not a big deal having two crates; the one we have works pretty well

martin: are there features in the C API that are not yet in our high level bindings?

nico: I wonder the fact they haven't updated their bindings, it means the API hasn't changed a lot recently

manish: harfbuzz rarely changes the API

martin: would it make sense to split?

manish: I'd have it in the servo org, the harfbuzz org or in its own org

manish: having an user in tree is useful

martin: other projects (like freetype-sys) don't have it

manish: that's a different case, and it's only used in Servo

> martin: I was referring to the one by ??? developers

Author: @mrobinson maybe you can fill the gap here? :innocent:

Reviewer: ```suggestion
martin: I was referring to the one by PistonDevelopers
```

gregory: We have patterns in the code, but GitHub Copilot is not able to recognize them. In the short term if it's experienced as spam, then you could have a temporary ban. With Stack Overflow they have done that.

rego: People can always use these tools and you may not notice that. If you have a nice editor that autocompletes, that's not a problem. It's these kind of PRs. If people are fine, maybe we can draft this proposal.

### Crowdfunding

rego: We still have the LFX platform open. We are still talking to the LF to transfer the money to open collective. That hasn't happened yet, but OC and GitHub sponsors have a bunch now. In a month or so, 2500 collected. We can talk about how to use this money in the future. Just an update on the situation.

### Outreach

rego: There is a GOSIM event happening in the Netherland in one week. Gregory will be talking about Servo there. Nico & Jonathan will Servo there.
gregory: It's next week.

rego: https://europe2024.gosim.org/

rego: The next day is RustNL. Rakhi is also attending that one. There will be a bunch of Servo people around and Rust people.

rego: https://2024.rustnl.org/

> rego: In a month there is the Web Engines Hackfest in June. Registration is still open. Nico and the Igalia team will be there. Registration is still open.

Reviewer 2: nit: last sentence is duplicate, we can remove it

Author: ```suggestion
rego: In a month there is the Web Engines Hackfest in June. Nico and the Igalia team will be there. Registration is still open.
```
